os/drivers/lcd/mipi_lcd.c

```diff
@@ -309,9 +309,10 @@ static int lcd_getplaneinfo(FAR struct lcd_dev_s *dev, unsigned int planeno, FAR
  * Name:  lcd_getpower
  ****************************************************************************/
 
-static int lcd_getpower(FAR struct lcd_dev_s *dev, int color)
+static int lcd_getpower(FAR struct lcd_dev_s *dev)
 {
-	return OK;
+	FAR struct mipi_lcd_dev_s *priv = (FAR struct mipi_lcd_dev_s *)dev;
+	return priv->power;
```

Comment:
Maybe SVACE says missing null check for the priv.

Reply:
@namanjain7 @anjana348 Could you check this?

Reply:
ok